### ISSUE TITLE
[Feature Request]: Add a quit shortcut ⌘ ,Q

### ISSUE THREAD

**Issue by ziarv** [beyondcode/herd-community#164]
### Feature Description

please add quit shortcut ⌘ ,Q

**Comment by sschlein** [comment]
Thanks for the suggestion but this does not work because is a background app that never gets the keyboard focus to actually receive the quit event. It's also designed to run permanently in the background and therefore we're not adding this soon.
